en/docs/samples/microprofile-jwt-sample.md

```diff
@@ -0,0 +1,136 @@
+# Configuring Microprofile JWT
+
+This page guides you through configuring [microprofile JWT (MP-JWT)](insertlink) for an OAuth/OpenID Connect web application. This is demonstrated using a **sample application** and describes how to generate JWT tokens by invoking the endpoints.
+
+----
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../guides/access-delegation/microprofile-jwt" target="_blank" rel="nofollow noopener">I have my own application</a>
+
+----
+
+
+## Set up the sample
+
+!!! note "Pre-requisites" 
+    Before running the samples, make sure you have [maven
+    3.x](https://maven.apache.org/download.cgi) installed.
+    
+
+First, let's configure the sample. Follow the steps given below:
+
+1.  Navigate to `<SAMPLE_HOME>/microprofile/microprofile-jwt/src/main/liberty/config` and open `server.xml` .
+
+2.  Replace `${CARBON_HOME}` with the directory where WSO2 Identity Server is installed on your machine.
```

Comment:
Aren't we supposed to get rid of CARBON_HOME?

Reply:
that's the placeholder given in the sample server.xml file